newIDE/app/src/EventsSheet/EventsTree/InstructionsList.js

```diff
@@ -55,7 +55,9 @@ type Props = {
 
   globalObjectsContainer: gdObjectsContainer,
   objectsContainer: gdObjectsContainer,
-};
+
+  rowIndex: number,
```

Comment:
I'm a tiny bit uneasy with this "rowIndex":

- It's name is very generic, so it's hard to grep/replace in the future.
- It's name gives an impression that this is mandatory and important. While it's "only" for an id for a selector.
- And the reader is not sure "what the heck is this row?". And why the instruction list even needs to know this information? 

I think we could have something like "idPrefix". And instead of passing a row number (which breaks the encapsulation by giving an information that is not useful to the child), the parent can pass `event-${rowIndex}`:

`<InstructionsList 
...
idPrefix={`event-${rowIndex}`}
/>`

and this prefix can be passed down.